packages/taco/test/conditions/compound-condition.test.ts

```diff
@@ -185,9 +196,9 @@ describe('validation', () => {
   ])('accepts shorthand for "%s" operator', (operator, operands, expected) => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const compoundConditionShorthands: Record<string, any> = {
-      'and': CompoundCondition.and,
-      'or': CompoundCondition.or,
-      'not': CompoundCondition.not,
+      and: CompoundCondition.and,
+      or: CompoundCondition.or,
+      not: CompoundCondition.not,
```

Comment:
When was this block changed to using quotes? I remember there was a recent change, in which I commented, with the same diff (i.e., removing the quotes). Is it possible that there's conflicting linting configurations somewhere?

Reply:
I think that's the case, but I wasn't able to reproduce that. I will keep an eye on that. The proper way to format is to use `pnpm lint:fix` and that should be consistent with the CI check, `pnpm ci:lint`.